rts/motoko-rts/src/stabilization/deserialization.rs

```diff
@@ -96,24 +97,16 @@ impl Deserialization {
             |context, field_address| {
                 *field_address = translate(context, *field_address);
             },
-            |context, slice_start, array| {
-                const SLICE_INCREMENT: usize = 127;
```

Comment:
Note the SLICE_INCREMENT in the slice_array is 128, not 127 (in case the change matters)

Reply:
Thanks for mentioning. I think the change is fine - it has no relevant impact (just a heuristic config, not too high, not too low - the value must be at least `TAG_ARRAY_SLICE_MIN` to be valid.)